timesketch/lib/analyzers/contrib/hashlookup_analyzer_test.py

```diff
@@ -43,19 +41,17 @@ def test_hash_match(self, mock_get_hash_info):
             message,
             ("Hashlookup Matches: 1"),
         )
-        mock_get_hash_info.assert_called_once()
+        mock_requests_get.assert_called_once()
```

Comment:
I'm not sure I can do that, I tried to do that with:
`mock_requests_get.assert_called_with('GET /lookup/md5/8ED4B4ED952526D89899E723F3488DE4 HTTP/1.1', 'Host: hashlookup.circl.lu')`

But I got:
~~~
E           AssertionError: expected call not found.
E           Expected: get('GET /lookup/md5/8ED4B4ED952526D89899E723F3488DE4 HTTP/1.1', 'Host: hashlookup.circl.lu')
E           Actual: get('blahsha256/ac7233de5daa4ab262e2e751028f56a7e9d5b9e724624c1d55e8b070d8c3cd09')
~~~

So it's look like I can only verify the url. Maybe I do it the wrong way, but I don't see how to do that.

Reply:
Yup - the mock only works at the function level, it has no idea what the actual HTTP request looks like. So if you're testing `requests.get(f"{self.hashlookup_url}sha256/{hash_value}")`

you want to use something like
```python
mock_requests_get.assert_called_with("http://hashlookup.circl.lu/lookup/sha256/ac7233de5daa4ab262e2e751028f56a7e9d5b9e724624c1d55e8b070d8c3cd09")
```

This will work if you set up your client with the right `self.hashlookup_url`. The idea is to say "given this initial set of parameters, this is what I expect the parameter of `requests.get()` to look like.

Reply:
Thanks for the clarification, it's fixed in f093665ceb10e2b2e257b4cdec1ccee471cbcee0.